symphonia-format-mkv/src/demuxer.rs

```diff
@@ -352,7 +352,9 @@ impl FormatReader for MkvReader {
                     segment_tracks = Some(it.read_element_data::<TracksElement>()?);
                 }
                 ElementType::Info => {
-                    info = Some(it.read_element_data::<InfoElement>()?);
+                    let raw_info = it.read_element_data::<InfoElement>()?;
+                    raw_info.copy_metadata_into(&mut metadata);
+                    info = Some(raw_info);
```

Comment:
Could you do something like this instead (since `info` is never `None`) to make it more concise?

```suggestion
                    info = Some(it.read_element_data::<InfoElement>()?);
                    info.as_ref().unwrap().copy_metadata_into(&mut metadata);
```